hack/test-go.sh

```diff
@@ -10,16 +22,15 @@ ${GO} vet --tags=test ./cmd/... ./pkg/...
 
 BASEDIR=$(pwd)
 
-echo "Installing golang staticcheck ..."
-GOBIN=${BASEDIR}/bin go install honnef.co/go/tools/cmd/staticcheck@latest
+#if static check true run this
+echo USE_STATIC_CHECK=$USE_STATIC_CHECK
```

Comment:
nit: you can skip this when you're done testing (i mean, I know that's what you're doing and all)